### ISSUE TITLE
Ajouter une logique de test

### ISSUE THREAD

**Issue by emilienschultz** [emilienschultz/pyshs-bib#27]

**Comment by emilienschultz** [comment]
ajout d'un fichier test_pyshs avec les premiers tests. @Carreau est-ce que la logique est bonne ?

**Comment by Carreau** [comment]
Oui, `test_pyshs.py` c'est bon, mais tu l'as appelé `tests_pyshs.py` (pluriel).

J'ai corrigé, etcorrigé les fixtures aussi, plus les `type(...) == ...` en `isinstance(..., ...)`.

Notons que pour lancer les tests, il faut juste faire `pytest` dans un shell, plutot que utiliser `if __name__ == "__main__"` pour multiple raisons.

**Comment by Carreau** [comment]
I would suggest for you to setup https://app.codecov.io/gh/emilienschultz/pyshs-bib ( I can't do it for you it's your account), it should give you a badge that tell you the percentage of the library that is tested.

I've setup github action to automatically tests the library on each pull request.

**Comment by emilienschultz** [comment]
I moved the tests in a test file : does this have a consequences on the automatization of the test ?
(where do you configure the test on github)
Thanks !!

**Comment by Carreau** [comment]
dans .github/workflows/tests.yml, mais ça devrais être bon, il va trouver
les tests tout seul.

**Comment by emilienschultz** [comment]
@Carreau je ne sais pas ce que j'ai cassé mais les tests échouent (ils passent sur ma machine). Une idée ?

**Comment by emilienschultz** [comment]
Nevermind, corrigé, mais je ne comprends pas pourquoi pytest passait en local vu qu'il y avait une erreur.
